apps/dotcms-ui-e2e/src/pages/DotSiteBrowser/DotSiteBrowser.ts

```diff
@@ -0,0 +1,66 @@
+import Form from '../../support/shared/Form';
+import Navigation from '../../support/shared/Navigation';
+import Page from '../../support/shared/Page';
+
+const URL = '/dotAdmin/#/c/site-browser';
+const CREATE_PAGE_BUTTON = '#addNewDropDownButtonDiv span.dijitButtonNode';
+const IFRAME = 'iframe#detailFrame';
+
+class DotSiteBrowser {
+    static openPage() {
+        cy.intercept('POST', 'BrowserAjax.openFolderContent.dwr').as('getLoggedinUser');
+        Navigation.visit(URL);
+        Navigation.assertPageUrlIs(URL);
+        cy.wait('@getLoggedinUser');
+    }
+
+    static checkSiteBrowserPageLoaded() {
+        Page.assertElementContains('.p-breadcrumb > ul > :nth-child(3)', `Browser`); // Header
```

Comment:
These selectors are very weak. Not sure what to do about it tho.

Reply:
agree, based on https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements

and what we started doing on core-web for unit testing, basically would be adding `data-testid` to every element needed

it might be too much though